src/datalad_installer.py

```diff
@@ -1473,6 +1473,10 @@ def install_package(
         extra_args: Optional[List[str]] = None,
         **kwargs: Any,
     ) -> Path:
+        if package == "git-annex" and platform.system() == "Windows":
+            raise MethodNotSupportedError(
+                "Conda does not support installing git-annex on Windows"
+            )
```

Comment:
I wonder if we should bother at this level to announce that since it would just fail right below anyways.
I would just keep it to the doc for this one unless some logic above already does some sensing on such an exception being thrown?

Reply:
The exception is handled on line 1000.  When `git-annex` is requested with `--method auto`, raising a `MethodNotSupportedError` will cause the script to try the next supported installation method after conda.